Code/Framework/AzCore/Platform/Common/Unimplemented/AzCore/Utils/Utils_Unimplemented.cpp

```diff
@@ -14,4 +14,20 @@ namespace AZ::Utils
     {
         return {};
     }
+
+    bool SetEnv(const char* envname, const char* envvalue, bool overwrite)
+    {
+        AZ_UNUSED(envname);
+        AZ_UNUSED(envvalue);
+        AZ_UNUSED(overwrite);
+
+        return false;
+    }
```

Comment:
I would use '[[maybe_unused]]' to be consistent
```suggestion
    bool SetEnv([[maybe_unused]] const char* envname, [[maybe_unused]] const char* envvalue, [[maybe_unused]] bool overwrite)
    {
        return false;
    }
```

Reply:
done